docs/reference_guides/model_libraries/generic/unit_models/pressure_changer.rst

```diff
@@ -28,12 +28,15 @@ Variable                    Name                  Notes
 =========================== ===================== ===================================================================================
 :math:`P_{ratio}`           ratioP
 :math:`V_t`                 volume                Only if has_rate_reactions = True, reference to control_volume.rate_reaction_extent
-:math:`W_{mechanical,t}`    work_mechanical       Reference to control_volume.work
+:math:`W_{mechanical,t}`    work_mechanical       Reference to control_volume.work :sup:`*`
 :math:`W_{fluid,t}`         work_fluid            Pump assumption only
 :math:`\eta_{pump,t}`       efficiency_pump       Pump assumption only
 :math:`W_{isentropic,t}`    work_isentropic       Isentropic assumption only
 :math:`\eta_{isentropic,t}` efficiency_isentropic Isentropic assumption only
 =========================== ===================== ===================================================================================
+:sup:`*` By default, ``control_volume.work`` is built only when the ``EnergyBalanceType`` of ``control_volume`` is `not` ``None`` and 
```

Comment:
You need to add a blank line here - this is causing a test failure due to an error in building the docs.

Reply:
Pushed a new commit to fix this.  Do we need click a button to run the checks?